### DISCUSSION TITLE
'Report' object has no attribute 'upload'

### DISCUSSION THREAD

**Discussion by marcosebarreto** [datapane/datapane#134] [Help] [answered]
Hi there. Does dp.Report work within Jupyter notebooks? Perhaps this question should be simple and already addressed here. but I'm getting this error ('Report' object has no attribute 'upload') when running some tutorial examples inside Jupyter notebooks. I am able to reproduce the examples when running from my terminal but not in Jupyter. Any tips? I'm using Datapane 0.11.9 and Python 3.8.5. Thanks.

**Comment by johnmicahreid** [comment, marked as answer]
Hi Marcose,

Datapane should work inside notebooks - it looks like your Jupyter kernel doesn't recognize the latest version of Datapane. Could you try restarting the kernel and/or restarting your Jupyter process?

**Reply by marcosebarreto** [reply]
Thanks John. It's working now. Best
